client/app/components/auth/sign-in.js

```diff
@@ -28,7 +28,9 @@ export default class AuthSignInComponent extends Component {
       isCityEmployee,
     } = await this.args.searchContacts(loginEmail);
 
-    if (isNycidEmailRegistered) {
+    // We should always be redirecting nyc government employees to the NYC.ID login page
+    // (never to the register page) regardless of whether isNycidEmailRegistered is true or false
+    if (isNycidEmailRegistered || isCityEmployee) {
```

Comment:
I'm wondering if this is adding complexity that will bite is later. Can't we assume that all city employees are registered and validated, and route them to NYC.ID login? 

I would add something about the if: 
```
if (isCityEmployee) {
  // transition to the auth.login route.
}
```

Reply:
We can't assume that all city employees are registered, because one of the government emails that I tested that was a City Employee was not registered in NYC ID. And my own government email was not considered "registered" until I went to NYC ID, signed in, and then clicked "Submit" on a "finish creating account" page that I posted in a screenshot above ^^. I can reach out to DOITT to confirm these assumptions before implementing this change. I only discovered this logic by testing the process myself. 

So you're saying that if a user is a City Employee we should never send them to the `auth.validate` route but ONLY send them to the `auth.login` route?

Reply:
Yeah — if it's an "unregistered" city employee, don't they just have to be directed to the login page?